packages/ra-ui-materialui/src/layout/SidebarToggleButton.tsx

```diff
@@ -29,11 +29,7 @@ export const SidebarToggleButton = (props: SidebarToggleButtonProps) => {
             )}
             enterDelay={500}
         >
-            <StyledIconButton
-                color="inherit"
-                onClick={() => setOpen(!open)}
-                size="large"
-            >
+            <StyledIconButton color="inherit" onClick={() => setOpen(!open)}>
```

Comment:
#8350 also suggested to pass the `...rest` props to the button, don't you think we should do it too?

Reply:
> #8350 also suggested to pass the `...rest` props to the button, don't you think we should do it too?

+1 on this

Reply:
I don't see the need for this